### ISSUE TITLE
The accessibility toggles on the top right cannot be tabbed to

### ISSUE THREAD

**Issue by ShanTulshi** [denysvitali/thebestmotherfuckingwebsite#48]
Using Chrome Version 84.0.4147.105

[These](https://github.com/denysvitali/thebestmotherfuckingwebsite/blob/5eb0a7bc1e496c8418580c473f3e7f9959f43d2f/src/js/main.fucking.js#L12) should probably be `button`s, not divs

**Comment by denysvitali** [comment]
Thanks @ShanTulshi , I'll fix this ASAP
